### PR TITLE
Add autoflake and isort to CI

### PR DESCRIPTION
**Pull request by evangriffiths**
Add isort and autoflake to CI, and run them on the codebase so CI passes.

### CODE REVIEW

**Comment by evangriffiths** [comment]
Lint on save working for me with this in my vscode settings.json
```
    "[python]": {
        "editor.insertSpaces": true,
        "editor.tabSize": 4,
        "editor.formatOnSave": true,
        "editor.codeActionsOnSave": {
            "source.organizeImports": "always",
            "source.unusedImports": "always",
        },
    },
```

in combination with the black, isort and autoflake vscode extensions. For some reason I couldn't get the same black formatting behaviour with `"python.formatting.provider": "black",` as when running black from the command line.

File: .github/workflows/python_ci.yaml [hunk 1]
```diff
@@ -50,3 +50,13 @@ jobs:
...
+
+  autoflake:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ./.github/actions/python_prepare
+      - name: Check with autoflake
+        run: |
+          poetry run autoflake --in-place --remove-all-unused-imports --remove-unused-variables --recursive .
+          git diff --exit-code --quiet || exit 1
```
[the remark this answers is not in the document]
Comment (kongzii) [reply .github/workflows/python_ci.yaml:62]: Why not just `git diff --exit-code` ? 🤔

**Review by kongzii — APPROVED** [review]

**Comment by kongzii** [comment]
Cool, will you add it to PMA too?

File: .github/workflows/python_ci.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (evangriffiths) [reply .github/workflows/python_ci.yaml:62]: Thanks for the answer coderrabbit 🤣

**Comment by evangriffiths** [comment]
> Cool, will you add it to PMA too?

Sure!

**Pull request merged**